### PR TITLE
chore: update email validation message

### PR DESCRIPTION
**Pull request by paulatulis**
#1206 
Updated error message to be in red text and reference the email domain that is blocked
Screenshot 2019-11-06 at 1 53 08 PM

### CODE REVIEW

File: src/components/Submit/index.tsx [hunk 1]
```diff
@@ -42,20 +42,35 @@ export const Submit: React.FunctionComponent<ISubmit> = ({ button, className, in
...
+          dangerouslySetInnerHTML={{
+            __html: response.success,
+          }}
+        />
       ) : (
         <div className="flex sm:flex-wrap sm:justify-center sm:py-4">
           <Input autoFocus {...input} value={value} onChange={setValue} onEnter={handleSubmit} />
 
-          <div className="flex-1 flex px-4 justify-end font-bold text-lg sm:justify-center sm:items-between sm:flex-wrap sm:py-2 sm:w-full sm:px-0">
+          <div className="px-4 justify-end font-bold text-lg sm:justify-center sm:items-between sm:flex-wrap sm:py-2 sm:w-full sm:px-0">
```
Comment (ssspear) [inline src/components/Submit/index.tsx]: We will still need `flex` in order for `justify-end` and the other flex properties to work. Here's a [link](https://tailwindcss.com/docs/justify-content) to the docs :)

File: src/hooks/useSubmitEmailForm.ts [hunk 2]
```diff
@@ -18,7 +18,12 @@ export const useSubmitEmailForm = (
   const submitForm = React.useCallback(
     (fields: IField[]) => {
       if (!validateEmail(fields[0].value)) {
-        return setResponse({ success: '', error: 'Please enter your work email' });
+        const ind: number = fields[0].value.split('').indexOf('@');
```
Comment (lottamus) [inline src/hooks/useSubmitEmailForm.ts]: I don't think you need to split on the empty string. `indexOf` should give you the correct character location in the string:

```suggestion
        const ind: number = fields[0].value.indexOf('@');
```

Or

```suggestion
        const blockedEmail = fields[0].value.split('@')[1];
```

File: src/hooks/useSubmitEmailForm.ts [hunk 3]
```diff
@@ -18,7 +18,12 @@ export const useSubmitEmailForm = (
   const submitForm = React.useCallback(
     (fields: IField[]) => {
       if (!validateEmail(fields[0].value)) {
-        return setResponse({ success: '', error: 'Please enter your work email' });
+        const blockedEmail = fields[0].value.split('@')[1];
+        const blockedEmail: string = fields[0].value.slice(ind);
```
Comment (lottamus) [inline src/hooks/useSubmitEmailForm.ts]: Got two `blockedEmail` variables in here now

```suggestion
```

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit fd6e8421bd7357ace454b2114518684d6dcc4660

https://deploy-preview-1220--stoplightio.netlify.com

**Pull request merged**